recipes/newrelic/infrastructure/ohi/postgres/debian.yml

```diff
@@ -71,6 +65,22 @@ install:
             exit 1
           fi
 
+    assert_pre_req:
+      cmds:
+        - |
+          # Map of tool names to the associated error code
+          required_tools_and_error_codes="date:41 sha256sum:42 base64:43 head:44"
```

Comment:
@NRhzhao I see we re using awk in the recipe, we should ensure that tool is installed.